metis-common/metis-common-mongo/src/main/java/eu/europeana/metis/mongo/connection/MongoProperties.java

```diff
@@ -80,12 +80,39 @@ public void setAllProperties(String[] hosts, int[] ports, String authenticationD
    * @param readPreferenceValue The read preference. Can be null, where then the default applies
    * @throws E In case either of the arrays is null, or their lengths don't match.
    */
+  @Deprecated
   public void setAllProperties(String[] hosts, int[] ports, String authenticationDatabase,
       String username, String password, boolean enableSsl, ReadPreferenceValue readPreferenceValue)
       throws E {
     setAllProperties(hosts, ports, authenticationDatabase, username, password);
     this.mongoEnableSsl = enableSsl;
     setReadPreferenceValue(readPreferenceValue);
+    setApplicationName("");
```

Comment:
This method should call the "new" method below with "null" applicationName parameter, as it is not it's overwriting the default value with the empty String

Reply:
Done